### DISCUSSION TITLE
Screenshr app stack definition

### DISCUSSION THREAD

**Discussion by filipemelo2002** [filipemelo2002/screenshr-app#11] [Polls]
We need to define the stack we will use so that we can get started on issue #1

**Comment by filipemelo2002** [comment]
Other options are welcome as well, please make sure to list them here and tag me in case you want  to put that up to vote ;)

**Reply by rennanprysthon** [reply]
I would recommend AstroJS because we could use different approaches on the same page. Some content could be server-side rendered, and others on the client, using React, for example.

If I had to choose one from the list above, I'd go with Angular or Next.js. I'd rather use a framework instead of building everything from scratch.

**Reply by filipemelo2002** [reply]
I don't really know AstroJS, and for the server-side rendering, Next.Js already implements it

**Reply by luisgomes01** [reply]
According to [astro doc](https://astro.build/) they are more performatic than nextjs. We have to keep in mind that the project isn't big though, the difference between them is probably minimal.
![image](https://github.com/filipemelo2002/screenshr-app/assets/72167417/1a0e74aa-53fb-41bb-b0bb-9af6b688b84c)

**Reply by rennanprysthon** [reply]
AstroJS has the island components concept. They can deliver server-side components and client-side on the same page
![image](https://github.com/filipemelo2002/screenshr-app/assets/45797445/4251773c-1a21-40d9-9321-bec1772f2d9c)

**Comment by rennanprysthon** [comment]
Also, we could check which backend tecnology we're going to use, and which database (or if we'll need one)

**Reply by filipemelo2002** [reply]
this discussion is out of the topic. We should have another repository to start brain storming these ideas
